spec/controllers/ops_controller/settings/common_spec.rb

```diff
@@ -231,6 +231,19 @@
           expect(edit[:new]).to eq(edit[:current])
         end
       end
+
+      context 'zone node' do
+        it 'sets ntp server info for display' do
+          controller.instance_variable_set(:@sb, :active_tab => 'settings_zone')
+          controller.instance_variable_set(:@edit, :new => {:ntp => {:server => ["zone1.pool.com", "zone2.pool.com"]}})
+          @zone = FactoryGirl.create(:zone)
```

Comment:
Does this need to be an `@` variable?  It won't be reused outside of the test context, right?